ddtrace/contrib/wsgi/wsgi.py

```diff
@@ -14,18 +14,6 @@
     from ddtrace import Tracer
     from ddtrace.settings import Config
 
-from ddtrace.internal.compat import PY2
```

Comment:
don't we still need this?

Reply:
I don't think so. I am not sure why we were importing `exceptions` and `builtins` in the first place. It seems unnecessary